style.css

```
@@ -0,0 +1,78 @@
+body {
+  display: flex;
+  flex-direction: column;
+  height: 700px;
+  background-color: #1c1a19;
+  background-image: url("Illustration.png");
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;

```

[OPTIONAL]

- [ ] Background images interferes with the text. You can try to give different `y` value to prevent this.
```suggestion

  background-position-y: 60vh;

```